bin/reth/src/node/events.rs

```diff
@@ -124,7 +127,7 @@ impl NodeState {
                 );
             }
             BeaconConsensusEngineEvent::CanonicalBlockAdded(block) => {
-                self.latest_canonical_engine_block = Some(block.number);
+                self.latest_block = Some(block.number);
```

Comment:
I think this needs to be moved to the `CanonicalChainCommitted` event ?

Reply:
makes sense, `CanonicalBlockAdded` is called on new payloads